tgui/packages/tgui/interfaces/GeneModder.js

```diff
@@ -0,0 +1,262 @@
+import { useBackend } from "../backend";
+import {
+  Button,
+  Section,
+  Flex,
+  Icon,
+  Collapsible,
+  LabeledList } from "../components";
+import {
+  ComplexModal,
+} from '../interfaces/common/ComplexModal';
+import { Window } from "../layouts";
+
+export const GeneModder = (props, context) => {
+  const { data } = useBackend(context);
+  const { has_seed } = data;
+
+  return (
+    <Window resizable>
+      <Window.Content display="flex" className="Layout__content--flexColumn">
```

Comment:
```suggestion
      <Window.Content className="Layout__content--flexColumn">
```
in-line styling uneeded since that is already defined in the SASS class
```css
.Layout__content--flexColumn {
  display: flex;
  flex-flow: column;
}
```